frontend/src/hooks/useActions.ts

```diff
@@ -588,4 +522,23 @@ const promptLoadFile = (parse, onData, errorMessage = 'Failed to parse file') =>
   input.click()
 }
 
+// Takes in the IDs of states, comments, and transitions
+// Parameters also include  the current project and whether a template is being created
+// Outputs a CopyData to be copied or Template object to be created into a template
+export const selectionToCopyTemplate = (stateIds: number[], commentIds: number[], transitionIds: number[], project): CopyData => {
+  const selectedStates: AutomataState[] = project.states.filter(state => stateIds.includes(state.id))
+  const selectedComments: ProjectComment[] = project.comments.filter(comment => commentIds.includes(comment.id))
```

Comment:
```suggestion
  const selectedStates = project.states.filter(state => stateIds.includes(state.id))
  const selectedComments = project.comments.filter(comment => commentIds.includes(comment.id))
```
No need after previous suggestion